composeApp/src/commonMain/kotlin/io/github/drp08/studypal/screens/RegistrationScreen.kt

```diff
@@ -0,0 +1,238 @@
+package io.github.drp08.studypal.screens
+
+
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import cafe.adriel.voyager.core.screen.Screen
+
+
+object RegistrationScreen : Screen {
+    @Composable
+    override fun Content() {
+        var name by remember { mutableStateOf("") }
+        var workingHoursStart by remember { mutableStateOf("") }
+        var workingHoursEnd by remember { mutableStateOf("") }
+        var hoursPerDay by remember { mutableStateOf("") }
```

Comment:
Can you make this come from a ViewModel? This data isn't local to the composable, but it would be fetched from a different source.